lib/buckets/common.js

```diff
@@ -568,7 +568,14 @@ function paginationStream(opts, onRecord, done) {
     var totalRecordsSent = 0;
     var doneEarly = false;
 
-    log.debug('paginationStream starting');
+    log.debug({
+        vnodes: Object.keys(vnodes),
+        limit: limit,
+        order_by: opts.order_by,
+        delimiter: delimiter,
+        prefix: prefix
+    }, 'paginationStream starting');
+
```

Comment:
Some of these changes are not related to node v6 update right?

Reply:
Correct... but very little:

1. This log line - this was required when debugging req.params vs req.query changes from the restify upgrade, and I figured it was a good log line to keep in
2. configure.js also has the process.stderr stream change to fix the RequestCaptureStream which is also not related to node v6 AFAICT

Reply:
:+1: